hera_cal/lstbin.py

```diff
@@ -500,15 +498,15 @@ def config_lst_bin_files(data_files, filetype='uvh5', dlst=None, atol=1e-10, lst
     # get output file lsts
     file_lsts = [lst_grid[start_index:end_index][ntimes_per_file * i:ntimes_per_file * (i + 1)] for i in range(nfiles)]
 
-    return data_times, lst_grid, dlst, file_lsts, start_lst
+    return lst_grid, dlst, file_lsts, start_lst, lst_arrays, time_arrays
 
 
-def lst_bin_files(data_files, filetype='uvh5', input_cals=None, dlst=None, verbose=True, ntimes_per_file=60,
+def lst_bin_files(data_files, input_cals=None, dlst=None, verbose=True, ntimes_per_file=60,
                   file_ext="{type}.{time:7.5f}.uvh5", outdir=None, overwrite=False, history='', lst_start=0, 
                   fixed_lst_start=False, atol=1e-6, sig_clip=True, sigma=5.0, min_N=5, rephase=False,
-                  output_file_select=None, **kwargs):
+                  output_file_select=None, Nblgroups=1, ignore_flags=False, **kwargs):
```

Comment:
I'd recommend making the default `Nblgroups=None` load all baselines simultaneously. That would match the convention in apply_cal and delay_filter and it's the only way to make loading everything straightforwardly accessible.

Reply:
Actually I think its easier to conceptualize the other way around. We know how much memory an entire file will take up (if its 5 GB on disk it will expand to roughly 2x that in memory based on `pyuvdata` performance). Knowing that I can only handle, say 1 GB of memory per task, its easy to hit that benchmark by specifying `Nblgroups=5`. Otherwise, I'd have to know exactly how many baselines were in the file to begin with, and then divide that by 5

Reply:
I can see your point. It would be nice to be consistent, but maybe it's not worth changing things.